led_strip/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.4.0
+- Support SPI mode `CONFIG_BLINK_LED_SPI`
```

Comment:
this config is not useable to the users. In fact, the SPI mode is always supported, they don't need to enable any Kconfig flags.

Reply:
also please don't forget to bump the version in the idf_component.yml